Language-Version-History.md

```diff
@@ -1,6 +1,6 @@
 # Features Added in VB Language Versions
 
-*NOTE:* See how to [specify the language version for your project here](https://docs.microsoft.com/en-us/dotnet/visual-basic/language-reference/configure-language-version)
+*NOTE:* See how to [specify the language version for your project here](/dotnet/visual-basic/language-reference/configure-language-version)
```

Comment:
@BillWagner Do you take these sorts of docs-platform specific changes in the C# spec repo?